costmap_resolution: 0.2
costmap_width: 300
costmap_height: 300
height_threshold_min: -1.3
height_threshold_max: 0.0
